public/index.html

```diff
@@ -33,6 +33,7 @@
       content="index, follow, max-image-preview:large, max-snippet:-1, max-video-preview:-1"
     />
     <meta property="og:type" content="website" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
```

Comment:
This sets the width to the width of the device and makes sure the scale is 1.0 (so it doesn't looked zoomed in or out).